lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -115,6 +120,17 @@ export abstract class ClientApplication {
         this.tokenCache = new TokenCache(this.config, this.browserStorage, this.logger, this.browserCrypto);
     }
 
+    /**
+     * Initializer function to perform async startup tasks such as connecting to WAM extension
+     */
+    async initialize(): Promise<void> {
```

Comment:
We should align this with what we'll be doing for broker. @pkanher617

Reply:
Once broker is merged we will reconcile the differences. Since broker is 1p only, we may not include this here.